functions/lib/emails/cancel_event.py

```
@@ -0,0 +1,72 @@
+        if response.status_code != 200:
+            logger.error(
+                f"Failed to send cancellation email for orderId={order_id}, body={response.text}"
+            )
+            return False
+
+        logger.info(
+            f"Successfully sent cancellation email to {email} for order {order_id}"
+        )
+        return True
+
+    except Exception as e:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)